src/main/java/com/github/shyiko/mysql/binlog/json/JsonStringFormatter.java

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright 2013 Stanley Shyiko
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.shyiko.mysql.binlog.json;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import com.github.shyiko.mysql.binlog.event.deserialization.ColumnType;
+
+/**
+ * A {@link JsonValueHandler} implementation that creates a JSON string representation.
+ *
+ * @author <a href="mailto:rhauch@gmail.com">Randall Hauch</a>
+ */
+public class JsonStringFormatter implements JsonFormatter {
```

Comment:
Again, all of the methods in this class are `protected` so that they can be overridden and accessed if clients create their own customized subclass of `JsonStringFormatter`.